### ISSUE TITLE
Where is the option -f in bcftools reheader? 

### ISSUE THREAD

**Issue by Shicheng-Guo** [samtools/bcftools#1077]
Hi All, 

I found the option -f in bcftools reheader was disappeared in bcftools 1.9, right? 

Thanks. 

Shicheng

----------------------------
bcftools reheader [OPTIONS] file.vcf.gz
Modify header of VCF/BCF files, change sample names.

-f, --fai FILE
add to the header contig names and their lengths from the provided fasta index file (.fai). Lengths of existing contig lines will be updated and contig lines not present in the fai file will be removed
-h, --header FILE
new VCF header
-o, --output FILE
see Common Options
-s, --samples FILE
new sample names, one name per line, in the same order as they appear in the VCF file. Alternatively, only samples which need to be renamed can be listed as "old_name new_name\n" pairs separated by whitespaces, each on a separate line. If a sample name contains spaces, the spaces can be escaped using the backslash character, for example "Not\ a\ good\ sample\ name".
--threads INT
see Common Options
----------------------------

**Comment by pd3** [comment]
Why would you think so? It's still in the code https://github.com/samtools/bcftools/blob/develop/reheader.c#L642

Maybe you are running an older version of the program by mistake? Try with `bcftools --version`.

**Comment by hurleyLi** [comment]
I have the same problem
```
> bcftools --version
bcftools 1.9
Using htslib 1.9
```
error message:
```
reheader: unrecognized option '--fai'

About:   Modify header of VCF/BCF files, change sample names.
Usage:   bcftools reheader [OPTIONS] <in.vcf.gz>

Options:
    -h, --header <file>     new header
    -o, --output <file>     write output to a file [standard output]
    -s, --samples <file>    new sample names
        --threads <int>     number of extra compression threads (BCF only) [0]
```

**Comment by hurleyLi** [comment]
Here're the relevant lines in the reheader.c included in the 1.9 release. @pd3 
```
static void usage(args_t *args)
{
    fprintf(stderr, "\n");
    fprintf(stderr, "About:   Modify header of VCF/BCF files, change sample names.\n");
    fprintf(stderr, "Usage:   bcftools reheader [OPTIONS] <in.vcf.gz>\n");
    fprintf(stderr, "\n");
    fprintf(stderr, "Options:\n");
    fprintf(stderr, "    -h, --header <file>     new header\n");
    fprintf(stderr, "    -o, --output <file>     write output to a file [standard output]\n");
    fprintf(stderr, "    -s, --samples <file>    new sample names\n");
    fprintf(stderr, "        --threads <int>     number of extra compression threads (BCF only) [0]\n");
    fprintf(stderr, "\n");
    exit(1);
}
```

**Comment by pd3** [comment]
Ah, I understand the confusion now. The question implied that the option was there and then disappeared. However, it was added only after 1.9 was released by the commmit https://github.com/samtools/bcftools/commit/9fb878af1a262be8676e0bd754adb9b560d809e0. 

Please use the latest development version instead. Installation instructions can be found here http://samtools.github.io/bcftools/howtos/install.html.

**Comment by Shicheng-Guo** [comment]
Thanks. Yes. lastest version solved the problems. The page you mentioned works not well. I suggest the following commands:

```
git clone https://github.com/samtools/htslib.git
git clone https://github.com/samtools/bcftools.git
cd bcftools
make
```


> Ah, I understand the confusion now. The question implied that the option was there and then disappeared. However, it was added only after 1.9 was released by the commmit [9fb878a](https://github.com/samtools/bcftools/commit/9fb878af1a262be8676e0bd754adb9b560d809e0).
> 
> Please use the latest development version instead. Installation instructions can be found here http://samtools.github.io/bcftools/howtos/install.html.
